tests/core/CoreTest.py

```diff
@@ -489,14 +493,16 @@ def test_run_complex(self):
             [(BearD_NeedsC.name, self.section1.name, self.filedict1)])
 
         # For BearE -> BearA.
-        bear_a = get_next_instance(BearA,
-                                   (result.bear for result in results))
+        bear_a = get_next_instance(
+            BearA,
+            (bear.bear for bear in bear_e.dependency_results[BearA]))
```

Comment:
hm I should rather rename the first `bear` to `result`

Reply:
Same for the other occurrences.

Reply:
`bear.bear` does look a bit confusing :confused: